### ISSUE TITLE
basicConfig discrepancies

### ISSUE THREAD

**Issue by alexprengere** [microsoft/picologging#11]
I tested picologging as a drop-in replacement for logging (CPython 3.10), it looks really great! A few things I picked up:

* I opened a small [PR](https://github.com/microsoft/picologging/pull/10) to fix the README instruction regarding basicConfig()
* when running:
  ```python
  import picologging as logging
  logging.basicConfig()
  logging.warning("test")
  # expected: WARNING:root:test
  ```
  On standard logging, [basicConfig](https://docs.python.org/3/library/logging.html#logging.basicConfig) configures a StreamHandler at WARNING level. picologging does not seem to do that, as the logging.warning does not show in the console.

* when running:
  ```python
  import picologging as logging
  logging.getLogger()
  # raises ValueError: character U+96c240 is not in range [U+0000; U+10ffff]
  # expected: <RootLogger root (WARNING)>
  ```

* another small issue:
  ```python
  import picologging as logging
  logging.basicConfig()
  logging.basicConfig(force=True)
  # raises AttributeError: 'picologging.StreamHandler' object has no attribute 'close'
  ```

* finally, but this is expected, [logging.getLevelName](https://docs.python.org/3/library/logging.html#logging.getLevelName) is not implemented, so I patched it on my end to run the tests

**Comment by tonybaloney** [comment]
Brilliant feedback, thank you. I'll work through this list.

**Comment by tonybaloney** [comment]
Shipped 0.2.0 with all these issues fixed
